docs/jwst/extract_2d/main.rst

```diff
@@ -133,11 +207,13 @@ For NIRCam and NIRISS WFSS, the ``extract_2d`` step has three optional arguments
   ``wavelengthrange`` reference file.
 
 
-For NIRCam TSGRISM, the ``extract_2d`` step has two optional arguments:
-
-``--extract_orders``
-  list. The list of orders to extract. The default is taken from the ``wavelengthrange``
-  reference file.
+For NIRCam TSGRISM, the ``extract_2d`` step has one optional argument:
 
 ``--extract_height``
   int. The cross-dispersion size (in units of pixels) to extract.
+
+
+For NIRCam and NIRISS WFSS mode, the ``extract_2d`` step has one optional argument:
+
+  ``--wfss_extract_half_height``
+    int. The cross-dispersion half size, in pixels.
```

Comment:
Again, note that this only applies to sources tagged in the source catalog as point-like (based on the catalog entry "is_star" - boolean).

Reply:
will make that check and point this pout in the documentation

Reply:
The catalog I have lists `is_star` as float, not boolean, and all values are NaNs.
```
<Column name='is_star' dtype='float64' description='Flag indicating whether the source is a star' length=116>
nan
nan
nan
nan

```
Do you know if this column is computed in photutils or source_catalog?

Reply:
From `source_catalog`:
```
@lazyproperty
    def is_star(self):
        """
        Flag indicating whether the source is a star.

        2020.03.02: The JWST Photometry Working Group has not determined
        the criteria for this flag.
        """

        # TODO: need algorithm for this flag
        #is_star = np.random.randint(2, size=len(self.id))
        #return is_star.astype(bool)
        return self.null_column
```

This is where NaNs are returned.

Reply:
An algorithm was specified a month or two ago, but it hasn't been implemented yet. It will be Boolean when implemented, with a value of `True` for point sources.